src/Compilers/Core/Portable/Syntax/SyntaxList`1.cs

```diff
@@ -306,6 +306,50 @@ public SyntaxList<TNode> ReplaceRange(TNode nodeInList, IEnumerable<TNode> newNo
             }
         }
 
+        /// <summary>
+        /// Creates a new list with the specified range of nodes removed.
+        /// </summary>
+        /// <param name="index">The starting index of the range to remove.</param>
+        /// <param name="count">The number of nodes to remove.</param>
+        /// <param name="newNodes">The new nodes.</param>
+        public SyntaxList<TNode> ReplaceRange(int index, int count, IEnumerable<TNode> newNodes)
```

Comment:
this doesn't seem to match any other rolyn list helpers (or anything in .net).  I don't feel like this is necessary. And even if we were to provide it, it would be sufficient to just be an extension method.